torchvision/prototype/models/quantization/mobilenetv3.py

```diff
@@ -0,0 +1,100 @@
+import warnings
+from functools import partial
+from typing import Any, List, Optional, Union
+
+import torch
+from torchvision.transforms.functional import InterpolationMode
+
+from ....models.quantization.mobilenetv3 import (
+    InvertedResidualConfig,
+    QuantizableInvertedResidual,
+    QuantizableMobileNetV3,
+    _replace_relu,
+)
+from ...transforms.presets import ImageNetEval
+from .._api import Weights, WeightEntry
+from .._meta import _IMAGENET_CATEGORIES
+from ..mobilenetv3 import MobileNetV3LargeWeights, _mobilenet_v3_conf
+
+
+__all__ = [
+    "QuantizableMobileNetV3",
+    "QuantizedMobileNetV3LargeWeights",
+    "mobilenet_v3_large",
+]
+
+
+def _mobilenet_v3_model(
+    inverted_residual_setting: List[InvertedResidualConfig],
+    last_channel: int,
+    weights: Optional[Weights],
+    progress: bool,
+    quantize: bool,
+    **kwargs: Any,
+) -> QuantizableMobileNetV3:
+    if weights is not None:
+        kwargs["num_classes"] = len(weights.meta["categories"])
+        if "backend" in weights.meta:
+            kwargs["backend"] = weights.meta["backend"]
+    backend = kwargs.pop("backend", "qnnpack")
+
+    model = QuantizableMobileNetV3(inverted_residual_setting, last_channel, block=QuantizableInvertedResidual, **kwargs)
+    _replace_relu(model)
+
+    if quantize:
+        model.fuse_model()
+        model.qconfig = torch.quantization.get_default_qat_qconfig(backend)
+        torch.quantization.prepare_qat(model, inplace=True)
+
+    if weights is not None:
+        model.load_state_dict(weights.state_dict(progress=progress))
+
+    if quantize:
+        torch.quantization.convert(model, inplace=True)
+        model.eval()
```

Comment:
Check carefully lines 44-54 comparing to original. This is a simplification of the code.